moveit_core/collision_distance_field/src/collision_common_distance_field.cpp

```diff
@@ -151,7 +150,7 @@ void getBodySphereVisualizationMarkers(GroupStateRepresentationConstPtr& gsr, st
 
   // creating sphere marker
   visualization_msgs::Marker sphere_marker;
-  sphere_marker.header.frame_id = std::move(reference_frame);
```

Comment:
Is the `std::move` being removed because it is redundant? Or is there some other reason?